next-app/globals.css

```diff
@@ -20,3 +20,14 @@ html,
 body {
   overscroll-behavior: none;
 }
+
```

Comment:
I think the proper tailwind way to do this is by extending the default theme as per https://tailwindcss.com/docs/theme

Could you try adding something like the following on in your tailwind config? 

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  theme: {
    extend: {
      text: {
       'xl': '1.25rem'
        '3xl': '2rem'
      }
    }
  }
}
```